src/utils/styles.ts

```diff
@@ -52,10 +52,15 @@ export const parseStyle = <T extends RNStyle>(
                 ...acc,
                 ...(Object
                     .keys(value) as Array<keyof typeof value>)
-                    .reduce((acc, key) => ({
+                    .reduce((acc, key) => {
```

Comment:
1) You need to fix indentation, I'm using one tab, here I can see 2 spaces.
2) Can you skip creating `const`s? This is the most important loop responsible for parsing styles and creating const make hurt performance / memory. You can leave comment as it will be unclear what we're trying to achieve here